README.md

```diff
@@ -27,21 +27,34 @@ All data is loaded up into PostgreSQL, including the Descriptions which we pull
 down as
 [`COMMENTS`](https://www.postgresql.org/docs/current/static/sql-syntax.html).
 
-You can find the readmes from the Texas Ethics Commission added in this project,
+You can find the readmes from the Texas Ethics Commission added in this project here:
 
 * [Contribution Data Readme](./data/TEC_CF_CSV/ReadMe.txt)
 * [Lobby Data Readme](./data/TEC_LA_CSV/LobbyLAR-ReadMe.txt)
 * [Form 1295](./data/tec_docs/1295CertificatesCSVFormat.pdf)
 * [Texas PDSERF/Plus
 Electronic Reporting Format](./data/tec_docs/TX_ERF13_7.pdf)
 
-
 Links
 ====
 
 * [TEC data](https://www.ethics.state.tx.us/dfs/search_CF.htm).
 * [PDSERF](https://dba.stackexchange.com/a/207425/2639) format
 
+Installation
+====
+
+Requirements: PostgreSQL, git, curl
+
+Repo download and database setup (example in bash):
+```
+$ git clone https://github.com/EvanCarroll/db-Texas-Ethics-Commission.git
+$ cd ./db-Texas-Ethics-Commission
+$ ./scripts/download_and_extract.sh
+$ createdb tec
+$ psql -f ./runme.sql
```

Comment:
I think you're missing a `-d tec` there, but I wouldn't use `tec` because we install to that schema so `tec.tec` is a little confusing. You should instead use 

    createdb mydb;
    psql -d mydb -f ./runme.sql